roles/docker_build_tag_push/tasks/main.yaml

```diff
@@ -0,0 +1,129 @@
+---
+#
+# This role creates the docker webserver and database images, then tags and 
+# pushes the images to a private registry that is setup in another role.
+# 
+# Two files containing the are copied to the device under test.  Each Dockerfile
+# is modified specifically to pull down its own base OS, mariadb package, and
+# apache package.
+#    mariadb_cont.tgz
+#    web_cont_2.tgz
+#
+
+# Need to handle the cases of building the docker containers for each variant
+
+  - name: Fail if ansible_docker0.ipv4.address is not defined
+    fail:
+      msg: "ansible_docker0.ipv4.address it not defined!"
+    when: ansible_docker0.ipv4.address is not defined
+    run_once: true
+
+# Red Hat
+  - name: copy mariadb_cont.tgz
+    copy: 
+      src=roles/docker_build_tag_push/files/rhelah/mariadb_cont.tgz
+      dest=/root
+      owner=root
+      group=root
+      mode=0644
+    when: ansible_distribution == "RedHat"
+
+  - name: copy web_cont_2.tgz
+    copy: 
+      src=roles/docker_build_tag_push/files/rhelah/web_cont_2.tgz
+      dest=/root
+      owner=root
+      group=root
+      mode=0644
+    when: ansible_distribution == "RedHat"
```

Comment:
Since these files share the same attributes and conditionals, you can use `with_items` to loop:

```yaml
- name: copy files
  copy:
     src: "roles/docker_build_tag_push/files/rhelah/{{ item }}"
     dest=/root
     owner=root
     group=root
     mode=0644
  with_items:
    - mariadb_cont.tgz
    - web_cont_2.tgz
  when: ansible_distribution == "RedHat"
```